x-pack/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/add_policy_to_template_confirm_modal.js

```diff
@@ -21,9 +20,12 @@ import {
   EuiText,
 } from '@elastic/eui';
 import { toastNotifications } from 'ui/notify';
+
 import { addLifecyclePolicyToTemplate, loadIndexTemplates } from '../../../../services/api';
 import { showApiError } from '../../../../services/api_errors';
-export class AddPolicyToTemplateConfirmModalUi extends Component {
+import { LearnMoreLink } from '../../../components/learn_more_link';
+
+export class AddPolicyToTemplateConfirmModal extends Component {
```

Comment:
TODO: Confirm this looks correct in the UI.

Reply:
![image](https://user-images.githubusercontent.com/1238659/59351947-68294000-8cd4-11e9-9ad6-4ecfdee16947.png)